h2/stream.py

```diff
@@ -1048,10 +1051,12 @@ def _build_hdr_validation_flags(self, events):
             # TODO: Find any places where we don't emit anything, and emit
             # an internal event, so we can do away with this branch.
             is_trailer = False
+            is_response_header = False
```

Comment:
Can you extend the comment above to mention that we always emit an event for responses, too?